exporter/exporter.go

```diff
@@ -85,16 +85,17 @@ func LoadMeasuresAndValueSets(measures []byte, valueSets []byte) {
 
 // GenerateCat1 generates a cat1 xml string for export
 func GenerateCat1(patient []byte, startDate int64, endDate int64, qrdaVersion string, cmsCompatibility bool) string {
-	var status bool
-	result := generateCat1(patient, startDate, endDate, qrdaVersion, cmsCompatibility, &status)
-	if status {
-		return "Export Failed"
+	var failed bool
+	var errmsg string
+	result := generateCat1(patient, startDate, endDate, qrdaVersion, cmsCompatibility, &failed, &errmsg)
+	if failed {
+		return "Export Failed: " + errmsg
 	}
 	return result
```

Comment:
What will happen in this return statement if it panics in `generateCat1`? Concerned since `result` is never successfully initialized. Will it just be empty string?